### DISCUSSION TITLE
aliases

### DISCUSSION THREAD

**Discussion by rock-kobs** [Security-Onion-Solutions/securityonion#5197] [Ideas]
would it be possible to setup aliases for ip addresses on future update? aliases would make easy ip identification such as who this ip belongs to,  etc

**Comment by weslambert** [comment]
Are you referring to resolving IP addresses to hostnames, etc? Please see: https://github.com/Security-Onion-Solutions/securityonion/discussions/5153#discussioncomment-1202723.
